### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by apoorvadhanala**
Created menu and added user pictures to display on main page along with for account.

### CODE REVIEW

**Review by anthonyqzhu — APPROVED** [review]
LGTM

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -32,4 +30,78 @@ body {
...
+.dropdown:hover .dropbtn {background-color: rgb(205, 236, 232);}
+
+.top-container {
+  display: inline-block; 
+  width: 100%;
+}
+
+.image-container{
+    display:flex;
+    align-items: center;
```
Comment (yjennywang) [inline src/main/webapp/style.css:95]: indentation?

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-    </div>
+    <head>
+        <meta charset="UTF-8">
+        <title>Welcome Page</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Cinzel">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lora">
+    </head>
+    <body style="background-color: rgb(205, 236, 232);">
```
Comment (yjennywang) [inline src/main/webapp/index.html:11]: Would it be better to keep this in the stylesheet? Presumably you want the same background color on every page.

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
+        <div id="content">
+            <br></br><br></br><br></br>
```
Comment (yjennywang) [inline src/main/webapp/index.html:13]: Use margin: or padding: on .title.

**Pull request merged**